static/js/vue-cdr-access/src/mixins/errorUtils.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+export default {
+    data() {
+        return {
+            show_404: false,
+            show_503: false,
+        }
+    },
+
+    methods: {
+        /**
+         * An empty response with a 200 return code signals that the uuid used in a query is valid, but not in the DCR.
```

Comment:
Huh, this is really weird, I don't think this is intentional. I've created followup ticket to look into why we're getting 200 responses, my reading of the FullRecordController is that its supposed to either fail on the access control check or throw a NotFoundException, so maybe the error handling isn't working as expected
https://jira.lib.unc.edu/browse/BXC-4180